web/src/components/ChatBotWidget.tsx

```diff
@@ -18,7 +19,7 @@ const ChatBotWidget = (): ReactElement => {
   useEffect(() => {
     const script = document.createElement('script')
 
-    script.src = 'https://unpkg.com/@rasahq/rasa-chat'
+    script.src = `/rasa-widget/chatbot.js`
```

Comment:
Widget is released unter Apache 2.0 licence, so copying it to our MIT licenced project should be fine.